packit/bot_api.py

```diff
@@ -106,6 +105,8 @@ def sync_upstream_release_with_fedmsg(self, fedmsg: Dict):
         self.sync_upstream_release(
             package_config=package_config, version=version, dist_git_branch="master"
         )
+        return "Success!"
```

Comment:
Please, could you add the `-> str` to the method definition so we know we are returning string?

```
def sync_upstream_release_with_fedmsg(self, fedmsg: Dict) -> str:
```

Reply:
Or we can capture `INFO` logs and return them instead of this.

Reply:
The response from this method is actually being fed back to github: I wonder how github displays it.

Reply:
I agree with the info - that way we have more info of what is happening. The response could be found in github in the webhook settings https://github.com/container-images/rsyslog/settings/hooks/65809338